dynatrace/api/openpipeline/settings/processor.go

```diff
@@ -519,24 +520,32 @@ func (ep *ValueAssignment) Schema() map[string]*schema.Schema {
 			Description: "Strategy to assign a value",
 			Optional:    true,
 		},
+		"multi_value_constant": {
+			Type:        schema.TypeList,
+			Elem:        &schema.Schema{Type: schema.TypeString},
+			Description: "Strategy to assign a value",
```

Comment:
```suggestion
			Description: "The constant multi value to assign",
```

Reply:
I aligned with the spec here. Maybe it makes sense to mention that the type has to be set accordingly, so that this field can be used

Reply:
done 👍